openlabcmd/README.md

```diff
@@ -29,3 +29,58 @@ optional arguments:
   --nocolor      Enable the no color mode.
   --recover      Enable the auto recover mode.
 ```
+
+### ha
+openlab ha cluster management command.
+
+#### node
+
+CRUD for node
+
+* openlab ha node list
+* openlab ha node get
+  ```
+  usage: openlab ha node get [-h] name
+  positional arguments:
+    name        The node name.
+  ```
+* openlab ha node create
+  ```
+  usage: openlab ha node create [-h] --type {nodepool,zuul,zookeeper-only}
+                              --role {master,slave} --ip IP name
+
+  positional arguments:
+    name                  The new node name, it should be global unique.
+
+  optional arguments:
+    -h, --help            show this help message and exit
+    --type {nodepool,zuul,zookeeper-only}
+                          The new node type. Choose from 'nodepool', 'zuul' and
+                          'zookeeper-only'
+    --role {master,slave}
+                          The new node role. It should be either 'master' or
+                          'slave'.
+    --ip IP               The new node's public IP.
+  ```
+* openlab ha node set
+  ```
+  usage: openlab ha node set [-h] [--maintain MAINTAIN] [--role {master,slave}] name
+
+  positional arguments:
+    name                  The node name.
+
+  optional arguments:
+    -h, --help            show this help message and exit
+    --maintain MAINTAIN   Set the node to maintained status.
+    --role {master,slave}
+                          Update node role. It should be either 'master' or
```

Comment:
**_NOTE_**: Please add a WARN about role updating, we should not update role of node except emergency situations, because it will impact checking scope of HA monitor , HA monitor will check and update with built-in policy automatically